core/unit_test/CMakeLists.txt

```diff
@@ -25,6 +25,24 @@ TARGET_INCLUDE_DIRECTORIES(kokkos_gtest PUBLIC ${GTEST_SOURCE_DIR})
 #Gtest minimally requires C++11
 TARGET_COMPILE_FEATURES(kokkos_gtest PUBLIC cxx_std_11)
 
+#
+# Define Incremental Testing Feature Levels
+# Define Device name mappings (i.e. what comes after Kokkos:: for the ExecSpace)
+#
+
+SET(KOKKOS_CUDA_FEATURE_LEVEL 999)
+SET(KOKKOS_CUDA_NAME Cuda)
+SET(KOKKOS_HIP_FEATURE_LEVEL 5)
```

Comment:
Are we actually only at 5?

Reply:
Yes it looks good. We don't have 10 and 6, 7, 8, and 9 don't exist

Reply:
actually 6 and 8 exist I think you got everything to 9 though